pkg/services/object_manager/transformer/fmt.go

```diff
@@ -0,0 +1,97 @@
+package transformer
+
+import (
+	"crypto/ecdsa"
+
+	"github.com/nspcc-dev/neofs-api-go/pkg"
+	objectSDK "github.com/nspcc-dev/neofs-api-go/pkg/object"
+	"github.com/nspcc-dev/neofs-node/pkg/core/object"
+	"github.com/pkg/errors"
+)
+
+type formatter struct {
+	nextTarget ObjectTarget
```

Comment:
Pipeline with the moulder will look like `moulder` -> `formatter` -> `sizeLimiter` -> `formatter`?

Reply:
Just `transformer` -> `moulder`.